java-grupa8/homework-4/src/main/java/ro/siit/java8/CarModel.java

```diff
@@ -0,0 +1,42 @@
+package ro.siit.java8;
+
+/**
+ * @author Alin_Negrea
+ * Defines the CarModel class characteristics
+ * CarModel extends from CarManufacturer class and has additional characteristics
+ * model
+ * electriclMotor
+ * electricBatteries
+ * energyConsumption
+ */
+
+public class CarModel extends CarManufacturer {
```

Comment:
Verify the OOP question: CarModel is-a CarManufacturer vs CarModel has-a CarManufacturer. Which one is true?